test/proto/BUILD

```diff
@@ -12,3 +12,29 @@ envoy_proto_library(
     name = "helloworld_proto",
     srcs = [":helloworld.proto"],
 )
+
+envoy_proto_library(
+    name = "bookstore_proto",
+    srcs = [":bookstore.proto"],
+    deps = [
+        "@googleapis//:http_api_protos",
+        "@protobuf_bzl//:cc_wkt_protos",
```

Comment:
I still see a reference to @protobuf_bzl here, maybe I'm looking at a stale patch..